### DISCUSSION TITLE
如何跟室友相處

### DISCUSSION THREAD

**Discussion by asd99263fg** [1101-GS4719-C/Python-Community-GS4719-C#2016] [General]
有時候真的不知道怎麼跟室友相處，明明都在同一間房間但是幾乎都不會聊天，每天頂多講關燈或者要睡的時候才講話，這樣在房間做什麼都很不自在，有人有遇過類似的問題嗎

**Comment by cassy0201** [comment]
可以試試從有共同修的課開始聊，或是觀察一下對方的興趣愛好，感覺都很不錯

**Comment by 710388** [comment]
佛系就好

**Comment by poiu6fi** [comment]
我的室友原本就認識了，所以平常都會說一些垃圾話

**Comment by OHoln** [comment]
先從找找共同的興趣愛好開始吧，有了能時常一起做的事就能有很多共通話題，自然也就會慢慢變熟了

**Comment by 155Julia** [comment]
我認為你的情況感覺好像一群i人住在了一起XD，感覺可以從共同有經歷的事情跟大家聊天，比如要選課了，問問室友都選些什麼、若要出去買宵夜順便問問室友要不要吃等等。但有時候這種安靜不甘預他人也是有好處的，比較不會受到打擾就像外宿一樣！

**Comment by yubo0oo** [comment]
我的建議是做好自己
彼此尊重
多注意下生活中的細節
宿舍不是自己家
